app/Stacks.js

```diff
@@ -43,25 +50,37 @@ module.exports = class Stack {
 
     for(var i = 0; i < cards.length; i++){
       this.addCard(cards[i]);
+      $(cards[i]).droppable('disable');
     }
     this.toggleDraggable();
-    this.cascadeCards();
+    this.toggleDroppable();
```

Comment:
Why are lines 55 and 56 required? They both call ```this.toggleDroppable();``` in series, which seems either redundant or a smell for bad code underlying the ```toggleDroppable()``` implementation.